blockstack/lib/rpc.py

```diff
@@ -1411,6 +1432,11 @@ def _dispatch(self, method_name):
                     'GET': self.GET_getinfo,
                 },
             },
+            r'^/v1/meminfo$': {
```

Comment:
Was this just for debugging or you intend to leave this in production?

Reply:
I was going to leave it in production so we can hook it into or monitoring infrasturcture.